hokusai/cli/review_app.py

```diff
@@ -0,0 +1,61 @@
+from collections import OrderedDict
+import click
+import yaml
+
+import hokusai
+
+from hokusai.cli.base import base
+from hokusai.cli.staging import KUBE_CONTEXT
+from hokusai.lib.common import set_verbosity, CONTEXT_SETTINGS
+from hokusai.lib.config import config
+
+@base.group()
+def review_app(context_settings=CONTEXT_SETTINGS):
+  """Manage/Create review apps"""
+  pass
+
+@review_app.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('app_name', type=click.STRING)
+@click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
+@click.option('-sf', '--source-file', type=click.STRING, default='hokusai/staging.yml', help='Source deployment file')
+@click.option('-dn', '--destination-namespace', type=click.STRING, default=None, help='Target Namespace')
+def create(app_name, verbose, source_file, destination_namespace):
+  set_verbosity(verbose)
+  # copy staging yml file
+  if destination_namespace is None: destination_namespace = app_name
+  create_new_app_yaml(source_file, app_name, destination_namespace)
+
+@review_app.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('app_name', type=click.STRING)
+@click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
+def apply(app_name, verbose):
+  """Create the Kubernetes resources defined in ./hokusai/%s.yml""" % app_name
+  set_verbosity(verbose)
+  hokusai.k8s_create(KUBE_CONTEXT, app_name, app_name)
```

Comment:
Would use `hokusai.k8s_update` here (also  update the function k8s_update to accept `yaml_file_name` as a keyword argument)- this will then use `kubectl apply` so you can apply updates to the spec